terror/terror.go

```diff
@@ -57,6 +61,7 @@ const (
 const (
 	CodeIncompatibleDBFormat ErrCode = iota + 1
 	CodeNoDataForHandle
+	CodeKeyExists
```

Comment:
In case we can't find a proper error code for our own error.
Some terror is not intended to return to client, we check it internally.
And maybe someday we need to map terror to PostgressSQL error.